packages/shell/src/cli-run.ts

```diff
@@ -1,162 +1,5 @@
-import { getDatabaseURL, XataApiClient } from '@xata.io/client';
-import { generateFromContext } from '@xata.io/codegen';
-import chalk from 'chalk';
-import fetch from 'cross-fetch';
-import dotenv from 'dotenv';
-import { EventEmitter } from 'events';
-import { createWriteStream } from 'fs';
-import fs, { mkdir } from 'fs/promises';
-import ora from 'ora';
-import { homedir, tmpdir } from 'os';
-import path, { dirname } from 'path';
-import RJSON from 'relaxed-json';
-import repl from 'repl';
-import { fileURLToPath } from 'url';
-
-const __filename = fileURLToPath(import.meta.url);
-const __dirname = dirname(__filename);
-
 export async function run(options: { env?: string; databaseURL?: string; apiKey?: string; code?: string } = {}) {
```

Comment:
Do we still need this file?

Reply:
The arguments? Whatever you prefer. I think keeping it and throwing an error is fine.

Reply:
Sorry, no. I meant if the cli-run and cli both are needed. Both are right now throwing errors, but I guess only one is being executed (cli one) and the other one is left unused (cli-run).

Reply:
The reason I kept `cli-run` is because it is imported by `index.ts`, which is the file that is exported as a library. So I'm keeping both and one will fail when used as a CLI app and the other one will fail when used as a library.